Tasks/Baboshko/Task 5/task5_1.py

```diff
@@ -0,0 +1,9 @@
+#first variant
+sum = lambda x, y: x+y
+print(sum(int(input()), int(input())))
```

Comment:
it's good that you know lambda, but I would prefer the fifth appointment of this task implemented instead of it